packages/neuron-ui/src/widgets/Calendar/index.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState, useEffect, useMemo } from 'react'
+import { getMonthCalendar, useLocalNames, monthInRange, yearInRange, dateEqual, dayInRange } from './utils'
+import styles from './calendar.module.scss'
+
+interface Option {
+  value: number
+  title: string
+  selectable: boolean
+}
+const Selector = ({ options, onChange }: { options: Option[]; onChange: (option: Option) => void }) => (
+  <ol className={styles.calOptions}>
+    {options.map(option => (
+      <li key={option.value} role="presentation">
+        <button
+          type="button"
+          aria-label={option.title}
+          title={option.title}
+          role="menuitem"
+          onClick={() => onChange(option)}
+          disabled={!option.selectable}
+        >
+          {option.title}
+        </button>
+      </li>
+    ))}
+  </ol>
+)
+
+export interface CalendarProps {
+  value: Date | undefined
+  onChange: (value: Date) => void
+  minDate?: Date
+  maxDate?: Date
+}
+const Calendar: React.FC<CalendarProps> = ({ value, onChange, minDate = null, maxDate = null }) => {
+  const [year, setYear] = useState(new Date().getFullYear())
+  const [month, setMonth] = useState(new Date().getMonth() + 1)
+  const [status, setStatus] = useState<'year' | 'month' | 'date'>('date')
+
+  useEffect(() => {
+    setYear(value?.getFullYear() ?? new Date().getFullYear())
+    setMonth((value?.getMonth() ?? new Date().getMonth()) + 1)
+  }, [value])
+
+  const locale = useLocalNames()
+  const weeknames = useMemo(() => [...Array(7).keys()].map(_ => locale.dayNamesMin[(_ + locale.firstDayOfWeek) % 7]), [
+    locale,
+  ])
+  const monthname = locale.monthNames[month - 1]
+
+  const calendar = useMemo(() => getMonthCalendar(year, month), [year, month])
+  function disabledTime(date: Date) {
+    return !dayInRange(date, { minDate, maxDate })
+  }
+  const calendarTable = (
+    <table className={styles.calendarTable}>
+      <thead>
+        <tr>
+          {weeknames.map(weekname => (
+            <th className={styles.calTableHeader} scope="col" key={weekname}>
+              {weekname}
+            </th>
+          ))}
+        </tr>
+      </thead>
+      <tbody>
+        {calendar.map(week => (
+          <tr key={week[0].label}>
+            {week.map(date => (
+              <td key={`${date.month}${date.date}`}>
+                <button
+                  type="button"
+                  data-type="button"
+                  aria-label={date.label}
+                  aria-pressed={dateEqual(date.instance, value)}
+                  title={date.label}
+                  className={`
+                    ${styles.calDateItem}
+                    ${date.isToday ? 'today' : ''}
```

Comment:
`isToday` could be marked by `aria-*` too, as `aria-pressed`, besides, `aria` has its standard now, `attr-*` would be better for a custom attribute`

Ref: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques

Reply:
add `aria-current` property